heat/core/arithmetics.py

```diff
@@ -129,6 +161,79 @@ def _add(self, other):
 DNDarray.__radd__.__doc__ = add.__doc__
 
 
+def add_(t1: DNDarray, t2: Union[DNDarray, float]) -> DNDarray:
+    """
+    WILL BE REVISED LATER
```

Comment:
Here we can briefly recall that either `t1.split=t2.split` or `t2.split=None` is required